app/controllers/charges_controller.rb

```diff
@@ -33,11 +33,19 @@ def create
       cart.clear
 
       flash[:success] = "Your payment was successful and your order is placed."
-      redirect_to order_path(order)
+      notify_boss
+      redirect_to order_path(@order)
     end
 
   rescue Stripe::CardError => e
     flash[:error] = e.message
     redirect_to cart_path
   end
+
+  private
+
+  def notify_boss
+    client = Twilio::REST::Client.new ENV["twilio_account_sid"], ENV["twilio_auth_token"]
+    message = client.messages.create from: '17723245092', to: '17724183162', body: "You've received a $#{(@order.total)} order!"
```

Comment:
Useless assignment to variable - `message`.<br>Line is too long. [128/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.